### PR TITLE
chore: replace keeper methods #NTRN-268

### PR DESCRIPTION
**Pull request by joldie777**
Task: https://hadronlabs.atlassian.net/browse/NTRN-268

Related PRs:
https://github.com/neutron-org/neutron-sdk/pull/138
https://github.com/neutron-org/neutron-dev-contracts/pull/48
https://github.com/neutron-org/neutronjsplus/pull/33
https://github.com/neutron-org/neutron-integration-tests/pull/289

List of changes:

1. Replaced the use of ica keeper RegisterInterchainAccount, SendTx methods with msg server ones.

### CODE REVIEW

**Comment by joldie777** [comment]
Test run: https://github.com/neutron-org/neutron-tests/actions/runs/8940208757

File: x/interchaintxs/types/expected_keepers.go [hunk 1]
```diff
@@ -35,7 +34,12 @@ type ICAControllerKeeper interface {
 	GetActiveChannelID(ctx sdk.Context, connectionID, portID string) (string, bool)
 	GetInterchainAccountAddress(ctx sdk.Context, connectionID, portID string) (string, bool)
 	RegisterInterchainAccount(ctx sdk.Context, connectionID, owner, version string) error
```
Comment (swelf19) [inline x/interchaintxs/types/expected_keepers.go]: you moved the method to ICAControllerMsgServer, i think it's save to remove this one

**Comment by joldie777** [comment]
Test run: https://github.com/neutron-org/neutron-tests/actions/runs/8986944456

File: x/interchaintxs/types/expected_keepers.go (hunk 1 above)
Comment (joldie777) [reply x/interchaintxs/types/expected_keepers.go]: Good catch! Thanks.

Comment (joldie777) [reply x/interchaintxs/types/expected_keepers.go]: Method is removed

**Review by swelf19 — APPROVED** [review]

**Review by pr0n00gler — APPROVED** [review]

**Pull request merged**